starters/deno-oak-denodb/src/graphql/resolvers/resolvers.ts

```diff
@@ -1,5 +1,5 @@
-import { createTechnology, deleteTechnologyById, updateTechnology } from './mutationHandler.ts';
-import { getTechnologies, getTechnology } from './queryHandler.ts';
+import { createTechnology, deleteTechnologyById, updateTechnology } from './mutation_handler.ts';
+import { getTechnologies, getTechnology } from './query_handler.ts';
```

Comment:
I renamed from camelCase to snake_case to be consistent with the Deno [style guide](https://deno.land/manual@v1.28.3/references/contributing/style_guide).